From 07ba3ae5e7338e53c08891af874e386b3282765f Mon Sep 17 00:00:00 2001
From: Brittany Wald <brittany.wald@joyent.com>
Date: Tue, 5 Sep 2017 23:57:00 +0000
Subject: [PATCH] MANTA-3387 node-fash npm publish cruft Reviewed by: Kody A
 Kantor <kody.kantor@gmail.com> Approved by: Kody A Kantor
 <kody.kantor@gmail.com>

---
 .npmignore | 6 +++++-
 1 file changed, 5 insertions(+), 1 deletion(-)

diff --git a/.npmignore b/.npmignore
index a56a7ef..3025614 100644
--- a/.npmignore
+++ b/.npmignore
@@ -1,2 +1,6 @@
+deps
 node_modules
-
+test
+tools
+.gitmodules
+Makefile
-- 
2.21.0

